bear/main.py.in

```diff
@@ -262,7 +262,11 @@ def parse_exec_trace(filename):
 
     logging.debug('parse exec trace file: %s', filename)
     with open(filename, 'r') as handler:
-        entry = json.load(handler)
+        try:
+            entry = json.load(handler)
+        except ValueError:
+            logging.debug('unable to parse: %s', filename)
+            return Execution(pid='', cwd='', cmd='')
```

Comment:
if you build the kernel with `make V=1` it will print exactly what it's invoking (similar to the compilation db generated by BEAR. It looks like scripts/extract-cert is invoked twice:

```sh
  scripts/extract-cert "" certs/x509_certificate_list || ( rm certs/x509_certificate_list; exit 1)
  scripts/extract-cert "certs/signing_key.pem" certs/signing_key.x509 || ( rm certs/signing_key.x509; exit 1)
```
Do you think BEAR has trouble parsing one of these?